programs/svm-spoke/src/instructions/deposit.rs

```diff
@@ -139,7 +139,7 @@ pub fn _deposit_v3(
     Ok(())
 }
 
-pub fn deposit_v3(
+pub fn deposit(
     ctx: Context<DepositV3>,
```

Comment:
should we also rename the context to `Deposit` for consistency? This has been changed in `RequestSlowFill` and `ExecuteSlowRelayLeaf` contexts.

Reply:
for this and all other changes here i have a proposed [PR](https://github.com/across-protocol/contracts/pull/868) that removes all other remaining V3 refrences.